.github/workflows/ci.yml

```diff
@@ -95,6 +104,7 @@ jobs:
         bundle exec rake
       env:
         VIEW_COMPONENT_PATH: ../view_component
+        RAILS_VERSION: '~> 6.1.0'
```

Comment:
Is this change necessary?

If so, can it use the matrix version instead of a hardcoded version of Rails?

Reply:
The pvc task runs without a matrix, so we can't use a matrix value (unless we want to create a matrix for this task).   It seems to fall back to `main` as a RAILS_VERSION without it and with the updated lockfile I was getting errors about sprockets not being available.  Explicitly setting Rails to a 6.1.x version addressed that.

I'm open to alternate suggestions.

Reply:
Ah great point, I completely missed that. I didn't expand the diff up high-enough and thought it was in the same section. YAML 😄